src/window/mouse_manager.rs

```diff
@@ -69,40 +74,76 @@ struct TouchTrace {
 }
 
 pub struct MouseManager {
-    dragging: Option<String>,
+    drag_details: Option<DragDetails>,
```

Comment:
Renaming to drag_details obscures the difference between this field and the drag_position field. So coming to this code as somebody new, its not as immediately obvious why drag_details might have a value or not.

Maybe we could call it current_drag_details or active_drag_details?

Reply:
I kept `drag_details` , but renamed the other variables to `grid_position` and `window_position`, which better reflects what they mean.